src/parsers/manifest/metaplaylist/representation_index.ts

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright 2015 CANAL+ Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import objectAssign from "object-assign";
+import { ICustomError } from "../../../errors";
+import {
+  IBaseContentInfos,
+  IRepresentationIndex,
+  ISegment,
+} from "../../../manifest";
+
+/**
+ * The MetaRepresentationIndex is wrapper for all kind of indexes (dash, smooth, etc)
+ *
+ * Ut wraps methods from origin indexes, while taking into account of the offset induced
```

Comment:
It*

Reply:
No, I wanted to use `Ut`, which is the Swedish word for outside